torcharrow/functional.py

```diff
@@ -109,6 +109,34 @@ def __getattr__(op_name: str):
     setattr(sys.modules["torcharrow.functional"], op_name, wrapper)
     return wrapper
 
+### operations in for text domain
+def add_tokens(
+    input_col: Union[ListColumn, List[Union[int, str]]],
+    tokens: Union[ListColumn, List[Union[int, str]]],
+    begin: bool,
+) -> NumericalColumn:
+    """
+    Append or prepend a list of tokens/indices to a column. 
+    This is a common operation to add EOS and BOS tokens to text.
+
+    Parameters
+    ----------
+    input_col: Column that has a list of input tokens/indices
+    tokens: Column that has a list of tokens/indices to append or prepend
```

Comment:
should we just say `list of tokens/indices to append or prepend`?

Reply:
Sure just updated the docstring!